scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,218 @@
+                        cwd=default_base_path,
+                        error_msg="Android Project Generation Failure.")
+
+    run_android_command([target_android_project_path / ('gradlew'+ android_support.GRADLE_EXTENSION), f'assemble{build_config.title()}'],
+                        cwd=target_android_project_path,
+                        error_msg="Gradle Build Failure.")
+    
+    if deploy_to_device:
+        if not activity_name:
+            activity_name = f'{ctx.project_name}Activity'
+
+        if not org_name:
+            org_name = f'org.o3de.{ctx.project_name}'
+        android_support.deloy_to_android_device(target_android_project_path,
+                                build_config,
+                                android_sdk_home,
+                                org_name,
+                                activity_name)
+
+def export_source_android_parse_args(o3de_context: exp.O3DEScriptExportContext, export_config: command_utils.O3DEConfig):
+    parser = argparse.ArgumentParser(
+                    prog=f'o3de.py export-project -es {__file__}',
+                    description="Exports a project as standalone to the desired output directory with release layout. "
+                                "In order to use this script, the engine and project must be setup and registered beforehand. ",
+                    epilog=exp.CUSTOM_CMAKE_ARG_HELP_EPILOGUE,
+                    formatter_class=argparse.RawTextHelpFormatter,
+                    add_help=False
+    )
+    
+    eutil.create_common_export_params_and_config(parser, export_config)
+
+    parser.add_argument('-deploy', '--deploy-to-android',default=False,action='store_true',help='At completion of build, deploy to android device.')
+
+    parser.add_argument('-actname', '--activity-name', type=str, default=f'{o3de_context.project_name}Activity',
+                        help='The name of the activity to invoke when deploying and running the APK.')
+    
+    parser.add_argument('-orgname', '--org-name', type=str, default=f'org.o3de.{o3de_context.project_name}',
+                        help='The name of the organization identifier (i.e. org.o3de) used to construct the full activity name.')
+
+    # parser.add_argument('-out', '--android-output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Android Project and APK location.')
+
+    parser.add_argument('-B', '--build-dir', type=str, 
+                                          help=f"The location to write the android project scripts to. Default: '{android.DEFAULT_ANDROID_BUILD_FOLDER}'", 
+                                          default=android.DEFAULT_ANDROID_BUILD_FOLDER) 
+
+    
+    
+    parser.add_argument('-apkm', '--asset-mode', type=str, default='PAK', choices=['PAK', 'LOOSE'],
+                        help='Choose how O3DE will package project assets into the final APK file.')
```

Comment:
Make this argument consistent with the one in the [android-generate command](https://github.com/o3de/o3de/blob/032450b765d6e34239493cdf346853c197f9b57d/scripts/o3de/o3de/android.py#L579-L584)